trl/trainer/ppo_trainer.py

```diff
@@ -54,6 +54,7 @@ def __init__(
         tokenizer: Union[PreTrainedTokenizer, PreTrainedTokenizerFast],
         dataset: Union[torch.utils.data.Dataset, Dataset],
         data_collator=None,
+        **kwargs,
```

Comment:
Since there is only one we need to pass here we can do it explicitly:
```suggestion
        num_shared_layers=None,
```

Reply:
Then we can also remove all the kwargs filtering logic below :)

Reply:
Sounds perfect!